SuperBuild/External_python-requirements.cmake

```diff
@@ -0,0 +1,22 @@
+set(proj python-requirements)
```

Comment:
I would change the name here to make it clear what is being installed and to match the Slicer convention, so: `python-openlifu` and  `External_python-openlifu.cmake`

Use of the `requirements` term is usually for external projects that are installing dependencies for a thing, not the thing itself, ex:

https://github.com/Slicer/Slicer/blob/main/SuperBuild/External_python-numpy.cmake

vs

https://github.com/Slicer/Slicer/blob/main/SuperBuild/External_python-pythonqt-requirements.cmake

Reply:
Hmm I was thinking if there were any other python requirements then they could be included here as well. 

So what I have in mind is "all python dependencies that the extension SlicerOpenLIFU needs"

That could include a super specific version of numpy or itk or something like that, so possibly it can be more than just the python package `openlifu`

Reply:
In that case, perhaps `python-sliceropenlifu-requirements`